app-admin/etcd-wrapper/files/etcd-member.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=etcd (System Application Container)
+Documentation=https://github.com/coreos/etcd
+Conflicts=etcd.service
+Conflicts=etcd2.service
+
+[Service]
+Type=notify
+Restart=always
+RestartSec=10s
+LimitNOFILE=40000
+TimeoutStartSec=60
+Environment=ETCD_NAME=%m
+Environment=ETCD_IMG_USER=etcd
+Environment=ETCD_DATA_DIR=/var/lib/etcd
+Environment=ETCD_TAG=v2.3.7
```

Comment:
@philips would you prefer to have etcd v3 as default version here to spread that faster?